### ISSUE TITLE
Instance segmentation API

### ISSUE THREAD

**Issue by bhack** [tensorflow/models#2793]
What is the timing of the new [Instance segmentation API](http://presentations.cocodataset.org/Places17-GMRI.pdf)? Will it cover ops like https://github.com/tensorflow/tensorflow/issues/2169 too?

**Comment by cy89** [comment]
Is your question when any code might be released? Have you tried asking the authors of the paper you linked?

**Comment by bhack** [comment]
Yes see slide n. 34. I've not Alireza Fathi github account to mention him. Probably you have some internal reference to other members of Google Team G-RMI to expose a roadmap.

**Comment by afathi3** [comment]
We plan to release the instance segmentation API by the end of November or early December.
Right now we do not have an implementation of the unpooling op you pointed to as far as I know, but we will look into it.

**Comment by bhack** [comment]
Thank you Alireza for the update. Seems that it is really near. On segmentation we have also FC-Densenet waiting in https://github.com/tensorflow/models/pull/2308

**Comment by tatatodd** [comment]
Closing this out, since this is really a request for information, rather than a bug or feature request.

**Comment by bhack** [comment]
@tatodd Was a request for information about a feature pre-announce (istance segmentation API/models) and the reply could be still of public interest. But it is so near that we could close this.

**Comment by bhack** [comment]
@afathi3 Can you post a reference here when there will be some news? Also if the plan will be postponed. Thanks

**Comment by afathi3** [comment]
We are working on it. It is going through the final processing.

**Comment by bhack** [comment]
Thank you. Will It include recurrent approach like Recurrent Segmentation for Variable Computational Budgets or it is another Google team?

**Comment by afathi3** [comment]
That is a different project.

**Comment by EGuirado** [comment]
@afathi3, Is there news of the segmentation API publication? Thanks

**Comment by bhack** [comment]
@afathi3 Is it related to your Instance Embedding Transfer paper?

**Comment by bhack** [comment]
It could be interesting how this new API will interect with Keras counterparts that will land as upstream in TF keras namespace. /cc @ahundt for https://github.com/keras-team/keras/issues/6538 https://github.com/keras-team/keras-contrib/pull/202 https://github.com/keras-team/keras-contrib/pull/197

**Comment by ahundt** [comment]
@bhack Thanks for including me on this discussion, as I'm very interested in utilizing such an API. There is a more research oriented implementation in https://github.com/tensorflow/models/tree/master/research/object_detection as well.

**Comment by bhack** [comment]
Can you reopen this untill the API is publishe?. So that we have a issue where we will be notified.

**Comment by bhack** [comment]
Another user opened https://github.com/tensorflow/tensorflow/issues/16452 so we could track that issue.

**Comment by deaspo** [comment]
Also waiting on this request
